clang/lib/Basic/Attributes.cpp

```diff
@@ -153,12 +154,35 @@ std::string AttributeCommonInfo::getNormalizedFullName() const {
       normalizeName(getAttrName(), getScopeName(), getSyntax()));
 }
 
+static const llvm::StringMap<AttributeCommonInfo::Scope> ScopeMap = {
```

Comment:
No static constructor. Can just make this an array and rely on it being sorted?

Reply:
We dont need it to be sorted. I removed the static constructor and made it a `std::map` instead.